src/SmartComponents/BaselinesTable/BaselineTableKebab/BaselineTableKebab.js

```diff
@@ -14,11 +14,20 @@ class BaselineTableKebab extends Component {
             isOpen: false
         };
 
-        this.findBaselineUUID = this.findBaselineUUID.bind(this);
+        this.getBaselineData = this.getBaselineData.bind(this);
+        this.deleteBaselineData = this.deleteBaselineData.bind(this);
         this.onKebabToggle = this.onKebabToggle.bind(this);
     }
 
-    findBaselineUUID() {
+    deleteBaselineData() {
+        const { fullBaselineListData, fetchBaselines, deleteBaseline, baselineRowData } = this.props;
+        let baselineId = baselinesTableHelpers.findBaselineId(baselineRowData, fullBaselineListData);
+
+        deleteBaseline(baselineId);
+        fetchBaselines();
```

Comment:
This performs a full fetch which may not have a performance impact. The alternative would be removing the deleted baseline (assuming success from server) from the data structure.